pkgs/by-name/ep/epson_201310w/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  rpmextract,
+  autoreconfHook,
+  file,
+  libjpeg,
+  cups,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "epson_201310w";
+  version = "1.0.1";
+
+  src = fetchurl {
+    url = "https://download3.ebz.epson.net/dsc/f/03/00/15/64/77/6f825c323a9f4b8429e3f9f0f4e9c50b1b15f583/epson-inkjet-printer-201310w-${finalAttrs.version}-1.src.rpm";
+    hash = "sha256-596qMon5KmMHe7mh+ykdpWeazJOCGA6HmmtvdGheRmA=";
+  };
+
+  nativeBuildInputs = [
+    rpmextract
+    autoreconfHook
+    file
+  ];
+  buildInputs = [
+    libjpeg
+    cups
+  ];
+
+  unpackPhase = ''
```

Comment:
We have [to call `preUnpack` and `postUnpack`](https://github.com/NixOS/nixpkgs/blob/4faa5f5321320e49a78ae7848582f684d64783e9/doc/stdenv/stdenv.chapter.md#phases-sec-stdenv-phases) at the top and bottom of this phase.